cmd2/cmd2.py

```diff
@@ -3592,7 +3592,8 @@ def _initialize_history(self, hist_file):
         try:
             with open(hist_file, 'rb') as fobj:
                 history = pickle.load(fobj)
-        except (AttributeError, EOFError, FileNotFoundError, ImportError, IndexError, KeyError, pickle.UnpicklingError):
+        except (AttributeError, EOFError, FileNotFoundError, ImportError, IndexError, KeyError, ValueError,
+                pickle.UnpicklingError):
             # If any non-operating system error occurs when attempting to unpickle, just use an empty history
```

Comment:
Might want to change the comment to say:

> If any of these errors occur when attempting to unpickle, just use an empty history

It's slightly misleading as-is since `FileNotFoundError` is an OSErrror.

Reply:
Good call.  I'll reword that and resubmit later.